lib/ipmi_backend_utils.pm

```diff
@@ -262,6 +262,21 @@ sub adjust_for_ipmi_xen {
     }
 }
 
+# Set boot from pxe via ipmitool
+sub set_pxe_boot {
+    while (1) {
```

Comment:
Can this while 1 be replaced with while $timer < $time_out_value? In extreme conditions, eg ipmi nic is plugged off or ipmi is not functioning, this while 1 will take hours to let the job stop after max_job_time.

Reply:
init_backend before run any test is expected to run from openqa side, so if there was IPMI connection problem, any test module are not supposed to be run.

Reply:
Yes, I thought about that. It could be possible that the infra problem(ipmi especially network which does not very rarely happen) starts after that. While 1 will be a nightmare with that.